Goal: be compatible with OpenLDAP 2.4

- remove schemacheck
- indent syntax correction

Author: Mathieu Parent <math.parent@gmail.com>

Upstream status: Already included, will be in 2.2.1

Index: b/templates/slapd.conf.template.in
===================================================================
--- a/templates/slapd.conf.template.in
+++ b/templates/slapd.conf.template.in
@@ -29,7 +29,8 @@
 replogfile      @ldapserver_replogfile@
 replicationinterval 5
 
-schemacheck 	       on
+#schemacheck doesn't exists in debian's slapd 2.4
+#schemacheck 	       on
 
 TLSCertificateFile     @sysconfdir@/kolab/cert.pem
 TLSCertificateKeyFile  @sysconfdir@/kolab/key.pem
@@ -49,9 +50,9 @@
 checkpoint 	512 5
 idlcachesize    10000
 idletimeout	300
-		# The idletimeout can be increased if some clients develop
-		# problems. Please report to kolab-devel@kolab.org
-		# if you encounter such a client.
+#		The idletimeout can be increased if some clients develop
+#		problems. Please report to kolab-devel@kolab.org
+#		if you encounter such a client.
 dirtyread
 
 directory	@ldapserver_dir@
